src/utils/base64.ts

```diff
@@ -14,14 +14,16 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+import Arweave from 'arweave';
```

Comment:
this import can always be tricky in a non web/node directory

Reply:
gotchya - still figuring out if this is the best way to do it, considering other options like https://www.npmjs.com/package/base64url at the moment.